spec/src/test/java/util/ObservableAssert.java

```diff
@@ -0,0 +1,32 @@
+package util;
+
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.ObjectAssert;
+import rx.Observable;
+
+import java.util.List;
+
+/**
+ * Assertion for an Observable.
+ * @param <T>
+ */
+public class ObservableAssert<T> extends AbstractAssert<ObservableAssert<T>, Observable<T>> {
+
+    public ObservableAssert(Observable<T> actual) {
+        super(actual, ObservableAssert.class);
+        isNotNull();
+    }
+
+    /**
+     * Verifies that the Observable receives a single non-null event and no errors.
+     * @return a new assertion object whose object under test is the received event
+     */
+    public ObjectAssert<T> hasNoErrors() {
```

Comment:
Observable<Void> is frequently used but this assertion would probably fail on the list size. Similarly observables where more than one item is emitted. Maybe change the naming to "hasExactlyOne" or two methods "hasNoErrors" for Observable<Void> and "hasExactly(n)" for observables actually emitting something. Maybe "hasNoErrors()" for Observable<Void> and "hasNoErrors(n)" for matching exactly?